x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -344,9 +359,7 @@ export const buildEqlDataProviderOrFilter = (
         'signal.group.id',
```

Comment:
One more thing, can we update this to `kibana.alert.group.id` per the sync from yesterday? Thanks!